.github/workflows/ci.yml

```diff
@@ -1,91 +1,32 @@
-        uses: actions/checkout@v2
-
-      - name: Run ROS Industrial CI
-        uses: ros-industrial/industrial_ci@master
-        env: ${{matrix.env}}
-
-  clang-format:
-    strategy:
-      matrix:
-        env:
-          - {ROS_DISTRO: melodic, ROS_REPO: main, CLANG_FORMAT_CHECK: file}
-
-    runs-on: ubuntu-18.04
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Run ROS Industrial CI
-        uses: ros-industrial/industrial_ci@master
-        env: ${{matrix.env}}
-
-  clang-tidy:
+  build-and-test:
+    runs-on: ubuntu-20.04
     strategy:
-      matrix:
-        env:
-          - {ROS_DISTRO: melodic, ROS_REPO: main, ADDITIONAL_DEBS: "clang-tidy-9", TARGET_CMAKE_ARGS: "-DCMAKE_CXX_CLANG_TIDY=clang-tidy-9", TARGET_WORKSPACE: "$TARGET_REPO_PATH metapackage_dependencies.rosinstall", UPSTREAM_WORKSPACE: "upstream_dependencies.rosinstall"}
-
-    runs-on: ubuntu-18.04
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Run ROS Industrial CI
-        uses: ros-industrial/industrial_ci@master
-        env: ${{matrix.env}}
-
-  catkin_lint:
-    strategy:
-      matrix:
-        env:
-          - { ROS_DISTRO: melodic, ROS_REPO: main, CATKIN_LINT: pedantic, CATKIN_LINT_ARGS: "--config $TARGET_REPO_PATH/.catkin-lint", TARGET_WORKSPACE: "$TARGET_REPO_PATH metapackage_dependencies.rosinstall", UPSTREAM_WORKSPACE: "upstream_dependencies.rosinstall" }
-
-    runs-on: ubuntu-18.04
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Run ROS Industrial CI
-        uses: ros-industrial/industrial_ci@master
-        env: ${{matrix.env}}
-
-  arm64_build:
-    strategy:
-      matrix:
-        env:
-          - {DOCKER_IMAGE: "arm64v8/ros:melodic-perception", ADDITIONAL_DEBS: "build-essential python-catkin-tools python-pip", BEFORE_INIT: "echo -e This test should run on aarch64. It is running on: $(uname -p) && [[ $(uname -p) == aarch64 ]]", CMAKE_ARGS: "-Wall -Wextra -Wpedantic -Werror", TARGET_WORKSPACE: "$TARGET_REPO_PATH metapackage_dependencies.rosinstall", UPSTREAM_WORKSPACE: "upstream_dependencies.rosinstall"}
-
-    runs-on: ubuntu-18.04
+      fail-fast: false
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
+    - name: Set Environment Variables 
+      run: |
+        echo "name=LANG" >> $GITHUB_ENV
+
+    - name: Setup ROS 2
+      uses: ros-tooling/setup-ros@0.1.4
+      with:
+        required-ros-distributions: foxy
+    
+    - name: Run Tests
+      uses: ros-tooling/action-ros-ci@0.2.0
+      with:
+        target-ros2-distro: foxy
+        package-name: uwrt_mars_rover
 
-      - name: Install Qemu and Start Qemu Docker
-        shell: bash
-        run: |
-          sudo apt update
-          sudo apt install qemu-user-static
-          docker run --rm --privileged multiarch/qemu-user-static --reset --credential yes --persistent yes
```

Comment:
you're probably gonna need to add this back somewhere to get arm64 builds working